Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -282,6 +335,70 @@ public struct TLSConfiguration {
         self.applicationProtocols = applicationProtocols
         self.keyLogCallback = keyLogCallback
     }
+    
+    private init(cipherSuites: [NIOTLSCipher],
```

Comment:
I think it might be nicer, instead of duplicating the memberwise init, to instead implement this on top of the existing one and just overriding the cipher suites afterwards.

Reply:
Do you mean to add another parameter to the existing private init?  Something like?
```swift
private init(cipherSuites: String,
		   cipherSuiteValues: [NIOTLSCipher],
		    ...
			 )
```

Reply:
Yeah, and we can just pass the default string.